### PR TITLE
adding new fellow Ivan Prinko

### PR DESCRIPTION
**Pull request by IvaKamie**

### CODE REVIEW

**Comment by matthewfeickert** [comment]
Hi @IvaKamie, do we have a Fellow project associated with you yet? If not, can you work with your mentors to add the information provided here https://iris-hep.org/docs/fellow to this PR?

**Comment by rct225** [comment]
Hi @IvaKamie You need to follow the directions outlined below to add yourself to the IRIS-HEP Website.  You're currently missing the markdown file.

https://iris-hep.org/docs/fellow

**Review by IvaKamie — COMMENTED** [review]
creating IvaKamie.md

File: pages/fellows/IvaKamie.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  end: 2022-09-14
+photo: /assets/images/team/Ivan-Prinko.jpg
+institution: Kyiv Academic University
+e-mail: ivan.prinko@protonmail.com
+project_title: Developing a Jupiter Notebook for converting confocal microscopic images to 3D cell images and metrics calculation.
+project_goal:
+    The project's objective is to provide an interactive tool for evaluating confocal microscopic images of cells in order to determine their features, including volume, polarity, and cross-sectional area.
+mentors:
+  - Abhishek Biswas (Princeton University)
+proposal: /assets/pdf/fellows-2022/<find-your-file>
```
Comment (matthewfeickert) [inline pages/fellows/IvaKamie.md:20]: @IvaKamie this needs to be an actual file path. Can you open up another PR that corrects this to your proposal?

**Pull request merged**